package.json

```diff
@@ -128,10 +128,10 @@
     "@types/uuid": "^3.4.0",
     "@types/winston": "^2.2.0",
     "@types/xml2js": "^0.4.0",
-    "electron": "1.7.11",
-    "electron-builder": "19.48.3",
+    "electron": "1.8.2",
```

Comment:
You need to bump `app/.npmrc` to this same version to ensure native node modules are built correctly.

Reply:
Yes, I spent about 30 mins figuring that out! 👍